src/Microsoft.Cci.Extensions/Extensions/AssemblyIdentityHelpers.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Cci.Extensions
+{
+    public static class AssemblyIdentityHelpers
+    {
+        public static string Format(this AssemblyIdentity assemblyIdentity)
+        {
+            var name = new System.Reflection.AssemblyName();
+            name.Name = assemblyIdentity.Name.Value;
+#if !COREFX
+            name.CultureInfo = new CultureInfo(assemblyIdentity.Culture);
```

Comment:
Some of these are going to potentially cause differences and maybe even bugs it is unclear looking at this in isolation.